client/ayon_houdini/plugins/publish/extract_hda.py

```diff
@@ -13,17 +12,33 @@ class ExtractHDA(plugin.HoudiniExtractorPlugin):
     families = ["hda"]
 
     def process(self, instance):
-        self.log.info(pformat(instance.data))
+
         hda_node = hou.node(instance.data.get("instance_node"))
         hda_def = hda_node.type().definition()
         hda_options = hda_def.options()
         hda_options.setSaveInitialParmsAndContents(True)
+        hda_options.setSaveSpareParms(True)
```

Comment:
```suggestion
        hda_options.setLockContents(True)
        hda_options.setSaveSpareParms(True)
```
Maybe trying to lock the contents might help to resolve the updating issue in the HDA.